CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ option(BUILD_OSTREE "Set to ON to compile with ostree support" ON)
 option(ALLOW_MANUAL_ROLLBACK "Set to ON to build with support of manual rollbacks" OFF)
 option(BUILD_AKLITE_OFFLINE "Set to ON to build cli command for an offline update" OFF)
 option(BUILD_AKLITE_WITH_NERDCTL "Set to ON to build with support of nerdctl/containerd" OFF)
-option(BUILD_AKLITE_APPS "Set to ON to build cli command for Apps management" ON)
 option(BUILD_TUFCTL "Set to ON to build sample tuf control application" OFF)
```

Comment:
The commit message is inaccurate. The next commit turns on composectl

Reply:
Why do you think it is inaccurate?

Reply:
Yes, it is.

Reply:
The commit 1e05cd3 only removes aklite apps. It does not turn on composectl usage.

Reply:
I agreed on it.

Reply:
Updated the commit message.